kubeflow/xgboost-operator
# XBoost Operator with Iris data classification example

## what is proposed in this PR 
we demonstrate the XGB operator with IRIS data classification example. 

## how to test this PR ? 
Local test, and see the test results.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/kubeflow/xgboost-operator/27)
